pkgs/development/python-modules/djangorestframework/default.nix

```diff
@@ -24,6 +25,14 @@ buildPythonPackage rec {
     sha256 = "sha256-XmX6DZBZYzVCe72GERplAWt5jIjV/cYercZGb0pYjoc=";
   };
 
+  patches = [
+    # See https://github.com/encode/django-rest-framework/issues/8608
+    (fetchpatch {
+      name = "fix-django-collect-static";
+      url = "https://patch-diff.githubusercontent.com/raw/encode/django-rest-framework/pull/8591.patch";
```

Comment:
Please fetch an explicit commit, not a pull request reference.

Reply:
:+1: